src/ophyd_async/panda/panda.py

```diff
@@ -116,6 +138,75 @@ def _block_name_number(block_name: str) -> Tuple[str, Optional[int]]:
     return block_name, None
 
 
+def seq_table_from_rows(*rows: SeqTableRow):
```

Comment:
This is the method we will use for now for i22, as per https://github.com/DiamondLightSource/i22-bluesky/blob/b56865cb26a9cedbe7f4d2c688d269a758421fa6/src/i22_bluesky/panda/fly_scanning.py#L33-L54